subsys/bluetooth/common/log.h

```diff
@@ -80,6 +82,7 @@ const char *bt_addr_le_str_real(const bt_addr_le_t *addr);
 #define bt_hex(buf, len) log_strdup(bt_hex_real(buf, len))
 #define bt_addr_str(addr) log_strdup(bt_addr_str_real(addr))
 #define bt_addr_le_str(addr) log_strdup(bt_addr_le_str_real(addr))
+#define bt_uuid_str(uuid) log_strdup(bt_uuid_str_real(uuid))
```

Comment:
It is general comment, not only for this PR: The documentation is sooo missed here.
This is not obvious for me, just looking into macro name that this calls log_strdup. Moreover - it is not obvious that this is log only macro.

Reply:
@rakons The fact that it is a log only macro is exactly why I moved `bt_uuid_str` from the public header files into this internal log header file. These four have a comment above that applies to them all.

The public API call `bt_uuid_to_str` has documentation. Is it satisfactory?

Is your general comment that we don't have documentation for internal functions?

Reply:
@joerchan 
> Is your general comment that we don't have documentation for internal functions?

General comment was to all functions in bluetooth/common/log.h header.
Is is internal file?

From a broader perspective it is also a pain to edit anything deeper in a stack without a word of comment for internal functions. But this is rather not a place to discuss this problem so wide.

Reply:
@rakons Public header files are under `include/` if they are not they are internal.

Not saying documentation isn't needed or helpful. They are. But for this PR i'd prefer to just move the function so that their visibility becomes correct.